src/screens/ProfileScreen/ProfileScreen.tsx

```diff
@@ -9,7 +9,7 @@ import UploadImageModalView from '../../components/GalleryModal';
 import {AuthContext} from '../../context/AuthContext';
 import {ImagePickerResponse} from 'react-native-image-picker';
 import Strings from '../../i18n/en';
-
+import {getData} from  '../../hooks/dataStoreHook'
```

Comment:
Remove this if it is not being used anywhere in this component.

Reply:
will resolve in next commit .